app/src/main/java/org/oppia/android/app/walkthrough/WalkThroughVerticalScrollListener.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.android.app.walkthrough
+
+/** Listener for when an activity should hide and show ProgressBar And HeaderTextView. */
+interface WalkThroughVerticalScrollListener {
```

Comment:
`WalkthroughVerticalScrollListener`, as we are treating `Walkthrough` as a single word, the `t` should be small.

Reply:
Done